structs.go

```diff
@@ -115,6 +116,9 @@ type Session struct {
 	handlers     map[string][]*eventHandlerInstance
 	onceHandlers map[string][]*eventHandlerInstance
 
+	httpInteractions   map[string]func(response *InteractionResponse) error
+	httpInteractionsMu sync.Mutex
```

Comment:
```suggestion
	HttpInteractions   map[string]func(response *InteractionResponse) error
	HttpInteractionsMu sync.Mutex
```

Reply:
I don't think this should be exposed as it's an interal